djangochannelsrestframework/mixins.py

```diff
@@ -19,6 +24,15 @@ class ListModelMixin:
     @action()
     def list(self, **kwargs):
         queryset = self.filter_queryset(self.get_queryset(**kwargs), **kwargs)
+
+        if hasattr(self, "paginate_queryset"):
```

Comment:
since `PaginatedModelListMixin` subclass the `List` mixin i think it would be nicer if it overrode the `list` method that way you will not need `hasattr(self, "paginate_queryset"):`